app/main/views/frameworks.py

```diff
@@ -293,6 +294,61 @@ def framework_submission_lots(framework_slug):
     ), 200
 
 
+@main.route('/frameworks/<framework_slug>/submissions/service-type', methods=['GET', 'POST'])
+@login_required
+@EnsureApplicationCompanyDetailsHaveBeenConfirmed(data_api_client)
+def choose_draft_service_lot(framework_slug):
+    framework = get_framework_or_404(data_api_client, framework_slug)
+
+    if framework['status'] not in ["open", "pending", "standstill"]:
+        abort(404)
```

Comment:
I think we only adding new services in open state, so couldn't change this to only allow access in open state?